public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,36 @@
+# dvc.api.get_url()
+
+Returns the full URL to the <abbr>data artifact</abbr> specified by its `path`
+in a `repo` (<abbr>DVC project</abbr>).
+
+## Signature
+
+```py
+get_url(path, repo=None, rev=None, remote=None)
+```
+
+## Parameters
+
+- **`path`** - used to specify the location of the target artifact within the
+  source project in `repo`, relative to the project's root.
+
+- `repo` - specifies the location of the source DVC project. Both HTTP and SSH
+  protocols are supported for online Git repositories (e.g.
+  `[user@]server:project.git`). `repo` can also be a local file system path to
+  an "offline" project.
+
+- `rev` - (optional)
+  [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
+  (such as a branch name, a tag, or a commit hash). This only works with `repo`
+  URLs.
+
+- `remote` - (optional) name of the [DVC remote](/doc/command-reference/remote)
+  to fetch the target artifact from
```

Comment:
Yet another question here @Suor: What if `repo` is a local project with no remotes setup. Can the API use the local cache as the data source? Similar to how `dvc get` works, per "url can also be a local file system path to an "offline" repository (in this case and if it doesn't have a default remote set up, instead of downloading, DVC will try to copy the target data from the external source project or its cache)." in https://dvc.org/doc/command-reference/get

Reply:
> Can the API use the local cache as the data source? 

I checked and it seems not (see 8b133caf). It makes sense for `get_url`, but what about `open`? Will try...